app/api/routes.py

```
@@ -475,4 +477,50 @@ def end_chat_session(request: SessionEndRequest, current_user: User = Depends(Au
+
+
+@router.get(
+    "/api/v1/consult/history/{session_id}",
+    tags=["에이전트 상담"],
+    summary="특정 상담 세션 상세 조회",
+    description="특정 세션에 대해 상담 로그를 시간순으로 조회합니다.",
+    response_model=list[ConsultMessage],
+)
+def get_consult_history_detail_routes(
+    session_id: str,
+    db: Session = Depends(get_db),
```

Ruff (0.14.5)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)